src/presenced.cpp

```diff
@@ -606,7 +606,6 @@ void Client::onChatsUpdate(::mega::MegaApi *api, ::mega::MegaTextChatList *rooms
                 {
                     uint64_t userid = peerList->getPeerHandle(j);
                     mChatMembers[chatid].insert(userid);
-                    addPeer(userid);
```

Comment:
Now that we don't need to track the peers of each chatroom, I think all the processing happening in `onChatsUpdate()` becomes obsolete and useless. That's why you removed the only purpose of the implementation of this callback: `addPeer()` and `removePeer()`.
I'll adjust it